boards/posix/native_posix/native_posix.dts

```diff
@@ -43,6 +43,11 @@
 		};
 	};
 
+	lvgl_pointer {
```

Comment:
ah, this changes the ordinals for native posix so you have to update https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/misc/check_init_priorities/validate_check_init_priorities_output.py#L10-L17 with the output you see in https://github.com/zephyrproject-rtos/zephyr/pull/62107/checks?check_run_id=16353533961

Reply:
Ahh, I was a bit confused why the test broke with seemingly unrelated errors :)

Reply:
yeah, may have to make the test a bit "smarter" if this become a hassle, though native_posix does not get changed often so maybe it's fine